test/parallel/test-buffer-isencoding.js

```diff
@@ -14,8 +14,8 @@ const assert = require('assert');
   'ucs-2',
   'utf16le',
   'utf-16le' ].forEach((enc) => {
-    assert.strictEqual(Buffer.isEncoding(enc), true);
-  });
+  assert.strictEqual(Buffer.isEncoding(enc), true);
+});
```

Comment:
this just looks weird. having the body of code on the same indentation level is confusing when quickly scanning the code.